buildpack.toml

```diff
@@ -48,6 +48,12 @@ api = "0.7"
     description = "whether to auto-configure Spring Boot environment properties from bindings"
     launch = true
     name = "BPL_SPRING_CLOUD_BINDINGS_DISABLED"
+  
+  [[metadata.configurations]]
+    build = true
+    default = "1"
```

Comment:
not needed anymore since the default should be retrieved from the `META-INF/MANIFEST.MF` ?

Reply:
Technically yes! But this is the 'happy path', see above comment on how this is the fallback value.